include/deal.II/base/index_set.h

```diff
@@ -608,11 +612,17 @@ class IndexSet
                     const bool     overlapping  = false) const;
 
 #  ifdef DEAL_II_TRILINOS_WITH_TPETRA
-  Tpetra::Map<int, types::signed_global_dof_index>
+  template <
+    typename NodeType =
+      LinearAlgebra::TpetraWrappers::TpetraTypes::NodeType<MemorySpace::Host>>
```

Comment:
why do you pick Host as the default here?

Reply:
We are using `MemorySpace::Host` as the default for all `Tpetra` wrappers. Without the default argument, we would need to adapt a couple of tests.